android/PartySampleNetworkCommon/lib/NetworkManager.cpp

```diff
@@ -169,10 +169,10 @@ NetworkManager::CreateAndConnectToNetwork(
     PartyNetworkConfiguration cfg = {};
 
     // Setup the network to allow the maximum number of single-device players of any device type
```

Comment:
is this comment now out of date? it's no longer the maximum number of single-device players of any device type

Reply:
this comment looks like it applies to your other edits as well